lib/grac/client.rb

```diff
@@ -29,6 +29,9 @@ def initialize(uri, options = {})
       [:params, :headers, :postprocessing, :middleware].each do |k|
         @options[k].freeze
       end
+      @patterns = @options[:postprocessing].keys.collect do |regex|
```

Comment:
By doing this in initialize `@patterns` is built every time a client is initialized.
Since grac makes heavy usage of method chaining, maybe we should instead keep initializing the `Regexp` objects in `postprocessing` but utilizing memoization